CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_netsuite_source v0.5.2
+
+## Features
+- Added this changelog to capture iterations of the package!
+- Added the `vendors_pass_through_columns` variable to be leveraged within the `stg_netsuite__vendors` model. This variable will allow users to pass through fields which are not natively included to the package staging model. This variable is empty by default and may be overridden within a users root `dbt_project.yml`.
+- Added the `items_pass_through_columns` variable to be leveraged within the `stg_netsuite__items` model. This variable will allow users to pass through fields which are not natively included to the package staging model. This variable is empty by default and may be overridden within a users root `dbt_project.yml`. 
```

Comment:
Can you link this PR in the CHANGELOG notes? See below for how we did it in past entries.

Reply:
Ooops! Yes, sure #23

Reply:
Ooops! Yes sure #23